CMakeLists.txt

```diff
@@ -104,6 +104,14 @@ foreach(version ${_SwiftFoundation_versions})
     endforeach()
 endforeach()
 
+# wasi-libc emulation feature flags
+set(_SwiftFoundation_wasi_libc_flags)
```

Comment:
nit: I think @etcwilde has mentioned to me before that you don't need to set a variable like this before appending to it, but perhaps I'm wrong on that

Reply:
While we can skip the explicit declaration, I usually prefer doing that for conditionally appended list variables to avoid reference to uninitialized CMake variables. A reference to uninitialized CMake variable is a little bit tricky and CMake provides [a way to warn them](https://cmake.org/cmake/help/latest/manual/cmake.1.html#cmdoption-cmake-warn-uninitialized), so I expect it's not a recommended way.